tool/teleport/common/teleport.go

```diff
@@ -970,32 +970,32 @@ func onConfigDump(flags dumpFlags) error {
 	entries, err := os.ReadDir(flags.DataDir)
 	if err != nil && !os.IsNotExist(err) {
 		fmt.Fprintf(
-			os.Stderr, "Could not check the contents of %s: %s\nThe data directory may contain existing cluster state.\n", flags.DataDir, err.Error())
+			os.Stderr, "\033[93mWARNING\033[39m: Could not check the contents of %s: %s\n         The data directory may contain existing cluster state.\n", flags.DataDir, err.Error())
```

Comment:
We should probably use `utils.Color` instead of hard-coding the control characters.

Reply:
Please see updates.